### PR TITLE
update icon: ubuntu (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #1740: [ICON REQUEST]: Ubuntu
### I have searched through the issues and didn't find my problem.

- [x] Confirm

### About the icon

one of the most used linux distribution

### Links and sources

[Ubuntu.fr](https://www.ubuntu-fr.org/)

### Additional information

I have seen the ubuntu-plain but i need the ubuntu-original

### PR DESCRIPTION
**Pull request by Gautam25Raj**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features
Update Icon

**This PR closes #1740**

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -9763,19 +9763,21 @@
...
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original",
+                "original-wordmark"
             ]
         },
-        "color": "#dd4814",
+        "color": "#e95420",
         "aliases": []
```
Comment (Panquesito7) [inline devicon.json:9781]: Aliases for `plain` and `plain-wordmark` should be added.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Gautam25Raj — COMMENTED** [review]
Update devicon.json and SVGs

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I think we still need a `plain-wordmark` version, as the `original-wordmark` version has two different colors (black and orange), which cannot be used in font versions.

**Comment by Gautam25Raj** [comment]
> I think we still need a `plain-wordmark` version, as the `original-wordmark` version has two different colors (black and orange), which cannot be used in font versions.

Okayy, will sure update them.

**Pull request closed without merging**